### DISCUSSION TITLE
小米hyperOS系统能用么?

### DISCUSSION THREAD

**Discussion by jzq135790** [ZCShou/GoGoGo#136] [General]
在之前一个版本发现无法使用.

**Comment by ZCShou** [comment]
这个你得自己测试一下

**Reply by jzq135790** [reply]
我试过了,还是能用的,不过微信里小程序好像不生效,共享实时位置也不行.高德地图进去确实位置变了.

**Reply by ZCShou** [reply]
> 我试过了,还是能用的,不过微信里小程序好像不生效,共享实时位置也不行.高德地图进去确实位置变了.

微信系列都不起作用是因为官方针对虚拟定位做了处理，屏蔽了虚拟定位。早期版本的微信是支持的，那时候微信朋友圈里到处是全球旅游的人，后来微信官方就屏蔽了
